### DISCUSSION TITLE
Next.js 12

### DISCUSSION THREAD

**Discussion by jsw** [serverless-nextjs/serverless-next.js#2013] [General]
Thanks for your great work on this project. I couldn't find any mention of next.js 12 support, and I'm not sure if there's any reason that 12 might not work with this component. If development or testing needs to happen, shall I open an issue so people can track progress?

**Comment by dphang** [comment]
Yeah, I need to update the docs and tests - just haven't had the chance yet. Plus I want to revamp docs once Lambda support is added (nearly ready for preview hopefully later this month). But I believe the latest component version (or alpha at least)  I did update it so it should at least not break deployments, but any new server side features haven't been looked at yet.
